tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -99,6 +101,37 @@ var _ = Describe("odo devfile create command tests", func() {
 			expectedFiles := []string{"package.json", "package-lock.json", "README.md", devfile}
 			Expect(helper.VerifyFilesExist(newContext, expectedFiles)).To(Equal(true))
 		})
+
+		It("should successfully create the devfile component and show json output for working cluster", func() {
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(devfilePath))
+			output := helper.CmdShouldPass("odo", "create", "nodejs", "--starter", "--context", newContext, "-o", "json")
```

Comment:
is there a reason for downloading the starter project in this test case? It is not getting used anywhere

Reply:
done, not downloading it anymore